spec/services/preview_service_spec.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+RSpec.describe PreviewService do
+  before do
+    stub_any_publishing_api_put_content
+    stub_any_asset_manager_call
+  end
+
+  describe "#create_preview" do
+    it "updates the Publishing API" do
+      edition = create(:edition)
+      request = stub_publishing_api_put_content(edition.content_id, {})
+      PreviewService.new(edition).create_preview
+      expect(request).to have_been_requested
+    end
+
+    it "marks the edition as revision synced" do
+      edition = create(:edition, revision_synced: false)
+      PreviewService.new(edition).create_preview
+      expect(edition.reload.revision_synced).to be(true)
+    end
+
+    it "cleans up draft assets" do
```

Comment:
There's a slight inconsistency between this and the 'try_create_preview' description. How about 'delegates cleaning up draft assets', just to make the content of the test more quickly identifiable?

Reply:
Sure